public/source/main.js

```diff
@@ -41,12 +41,9 @@ var adBlocker = require('just-detect-adblock');
       clearTimeout(id);
     };
 
-  programEvents.add(function(event) {
-    if (event.event === 'init-tooltip') {
-      $('.bootstrap-tooltip').tooltip().removeClass('bootstrap-tooltip');
```

Comment:
No need to remove the `bootstrap-tooltip` class anymore as the init process of new elements is handled by bootstrap and won't call it twice one existing tooltips

Reply:
Honestly, I think we can get rid of `bootstrap` as it's footprint has been reduced.

Reply:
@jung-kim I'm not sure what you mean. 
Bootstrap is still used all over the place with tooltips, modals, dropdowns, and general styling.